packages/pointer_interceptor/pointer_interceptor_web/example/README.md

```diff
@@ -7,3 +7,17 @@ package.
 
 Unless you are making changes to this implementation package, this example is
 very unlikely to be relevant.
+
+## Getting Started
+
+`flutter run -d chrome` to run the sample. You can tweak some code in the `lib/main.dart`, but be careful, changes there can break integration tests!
+
+## Running tests
+
+**Make sure that you have `chromedriver` running in port `4444`.**
+
+`flutter drive --target integration_test/widget_test.dart --driver test_driver/integration_test.dart --show-web-server-device -d web-server --web-renderer=canvaskit`
```

Comment:
Consider replacing all of this with a link to information about [`drive-examples`](https://github.com/flutter/packages/blob/main/script/tool/README.md#run-dart-integration-tests), and mentioning the `--web` and `--run-chromedriver` flags.

Reply:
Done!